src/App.css

```diff
@@ -0,0 +1,155 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+.background{
+    background-color: rgb(8, 9, 8);
+
+}
+
+/*akvmlkfvlkfvlkavkalvaklfkvlkqmnhnnnnnnnnnnnnnnnnnnnvlkq*/
+
+body{
+  background-color:#a03d3d;;
+  padding: 0px;
+  margin: 0px;
+  color: rgba(0,0,0,0.87);
+  font-family: 'Roboto', sans-serif;
+}
+
+.app{
+  text-align: center;
+  background-color: white;
+  height: auto;
+  min-height: 100vh;
+  width : 40%;
+  margin: auto;
+  padding : 30px;
+  box-shadow: 2px 2px 20px #ececec;
+}
+
+.app-title{
+  margin: 0;
+  margin-bottom: 30px;
+}
+
+.add-todo{
+  margin: 30px 0px;
+}
+
+.task-input{
```

Comment:
if you've created task related separate css file then you don't have to add that css here too.